src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -38,7 +38,14 @@ protected LoopingNgJob(ILoggerFactory loggerFactory)
                 }
 
                 Logger.LogInformation("Running job.");
-                await RunInternal(cancellationToken);
+                try
+                {
+                    await RunInternal(cancellationToken);
+                }
+                catch(TransientException te)
```

Comment:
IMO the try catch should be done inside RunInternal, without changing LoopingNgJob